crates/ruff/src/args.rs

```diff
@@ -443,27 +447,22 @@ pub struct FormatCommand {
     #[clap(long, overrides_with("preview"), hide = true)]
     no_preview: bool,
 
-    /// Format code starting at the given character offset (zero based).
+    /// Format code in the given <RANGE>.
     ///
-    /// When specified, Ruff will try to only format the code after the specified offset but
-    /// it might be necessary to extend the start backwards, e.g. to the start of the logical line.
+    /// The <RANGE> uses the format `<start_line>:<start_column>-<end_line><end_column>`.
```

Comment:
Is column a byte offset, codepoint offset or a grapheme cluster offset? :-)

(I think the "most correct" thing is a grapheme cluster offset... But this is another thorny question about what inputs will be produced. And this could likely come up, e.g., with emoji in strings. If an emoji has multiple codepoints in it and column numbers are counted by codepoint by Ruff but by grapheme cluster in the editor, that could lead to a mismatch.)

Reply:
I'll extend the documentation but I'll go with codepoint because that's what we get from VS Code.